docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Aug-29_netCDF_packing_changes.txt

```diff
@@ -0,0 +1,3 @@
+* A 'fill_value' key can no longer be specified as part of the `packing` argument to `iris.save` when saving in netCDF format. Instead, a fill value should be specified as a separate `fill_value` argument if required.
```

Comment:
Please add a note saying that multiple fill values may be supplied, and these are applied to output cubes in sequence.

Reply:
How I about I just add "or list of fill values"? I think we can assume that anyone familiar with the packing attribute knows that if you specify a list of values they applies to each cube in turn. I will modify the other what's new entry to explain what happens when you pass a list of fill values.